.github/workflows/build-test-prValidation.yml

```diff
@@ -8,17 +8,18 @@ concurrency:
   group: ci-${{ github.ref }}
   cancel-in-progress: true
 
-env:
-  CLOUDSMITH_MACHINE_AKKA_CREDS: ${{ secrets.CLOUDSMITH_MACHINE_AKKA_CREDS }}
-
 permissions:
   contents: read
 
 jobs:
   check-code-style:
     name: Check / Code Style
-    runs-on: ubuntu-22.04
+    runs-on: Default
```

Comment:
Could we name it something more akka specific? `Default` sounds like it is the default from github.

Reply:
`Akka-Default` maybe?